fbpic/particles/particles.py

```diff
@@ -371,15 +371,13 @@ def make_ionizable( self, element, target_species,
         if hasattr( self, 'int_sorting_buffer' ) is False and self.use_cuda:
             self.int_sorting_buffer = np.empty( self.Ntot, dtype=np.uint64 )
 
-    def handle_ionization( self ):
+    def handle_elementary_processes( self ):
         """
-        Ionize this species, and add new macroparticles to the target species
+        Handle elementary processes for this species (e.g. ionization)
         """
+        # Ionization
         if self.ionizer is not None:
-            if self.use_cuda:
-                self.ionizer.handle_ionization_gpu( self )
-            else:
-                self.ionizer.handle_ionization_cpu( self )
+            self.ionizer.handle_ionization( self )
```

Comment:
In the `compton_scattering` branch, there is another function call here, in order to handle Compton scattering.